app/recordtransfer/templates/recordtransfer/email/email_base.html

```diff
@@ -0,0 +1,104 @@
+{% load i18n %}
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="description"
+              content="Email communication from the National Centre for Truth and Reconciliation.">
+        <title>{% blocktrans %}Email from Our Company{% endblocktrans %}</title>
+    </head>
+    <body style="margin: 0;
```

Comment:
It looks like body tags aren't well supported. You could probably change this to a `<div>` without issue.

![body](https://github.com/user-attachments/assets/26e26692-7962-4f43-89a7-9d8d7dbbb010)